### ISSUE TITLE
[CONSOLE] show deployed protocol version

### ISSUE THREAD

**Issue by d10r** [superfluid-finance/superfluid-console#186]
With the 1.8 release, we get a versionString in Resolver which tells us what's deployed.
This version string includes the ethereum-contracts package version and the git revision (leading 16 digits).

The information is store in the Resolver under the key `versionString.v1`.
Since the Resolver only supports address values, it needs to be parsed like that: https://github.com/superfluid-finance/protocol-monorepo/blob/dev/packages/ethereum-contracts/ops-scripts/libs/common.js#L442

- Add a new tab for versions including
    - Protocol contracts
    - Peripheral contracts
    - Subgraph
    - Sentinels (compatible version number)

**Comment by d10r** [comment]
There's something wrong.
After upgrading the protocol contracts for polygon-mumbai, it's supposed to show 1.8.1-b182e1dac38615d4.
But keeps showing 1.8.1-b50c82f4a96d2425 in the console.

console link: https://console.superfluid.finance/mumbai/protocol

resolver: https://mumbai.polygonscan.com/address/0x2feFd8B23e4770B674b0d2fAda5EE8a2bE0B525F#readContract

![image](https://github.com/superfluid-finance/superfluid-console/assets/5479136/40d18a37-83fd-46b4-a36c-81744b9757e1)

![image](https://github.com/superfluid-finance/superfluid-console/assets/5479136/a7c0e74f-7d08-45eb-a519-04bb92038531)

**Comment by 0xdavinchee** [comment]
good catch @d10r if I had to guess it is because we've redeployed the resolvers + updated it in metadata.json, but we haven't released SDK-Core/Redux to use the new metadata or bumped the version used by the console.
